README.md

```diff
@@ -1,13 +1,47 @@
+![Azure DevOps builds](https://dev.azure.com/eugenypetlakh/serilog.extension/_apis/build/status/Softeq.serilog.extension?branchName=master)
+![NuGet](https://img.shields.io/nuget/v/Softeq.Serilog.Extension.svg)
+
 # Query Utils
 
 Query Utils extensions for IQueryable to perform Paging, Filer, Sorting
 
-## About
-
-This project is maintained by Softeq Development Corp.
+## Usage
+Query Utils has interfaces:
+- IPagedQuery - pagination results
```

Comment:
Need a more detailed description for every interface.

Reply:
Query Utils extension currently supports the following types of data manipulations, which you may implement in your query:
- IPagedQuery (allows you to get a sample of data spitted by pages)
...